dask_expr/io/io.py

```diff
@@ -50,6 +51,10 @@ def _layer(self):
 class BlockwiseIO(Blockwise, IO):
     _absorb_projections = False
 
+    @functools.cached_property
+    def _factor(self):
+        return 1
```

Comment:
This could probably use a better name, or an explanatory docstring

Reply:
Yeah good point